src/fileio_type.h

```diff
@@ -40,6 +40,8 @@ enum DetailedFileType {
 	DFT_FIOS_DIR,    ///< A directory entry.
 	DFT_FIOS_DIRECT, ///< Direct filename.
 
+	DFT_END,         ///< End of this enum. Supports a compile time size check against _fios_colours in fios_gui.cpp
```

Comment:
```suggestion
	DFT_END,         ///< End of this enum.
```

Reply:
"End of this enum."  isn't a useful comment.  It's an enum called DFT_END. The value's name is self descriptive.

The longer form comment I wrote explains what the value is for. Why it exists.